src/config.ts

```diff
@@ -0,0 +1,23 @@
+type Config = {
+  apiKey?: string;
+  authDomain?: string;
+  databaseURL?: string;
+  projectID?: string;
+  storageBucket?: string;
+  messagingSenderID?: string;
+  appID?: string;
+  apiBaseURL?: string;
```

Comment:
apiURL should be good

Reply:
Maybe I should just remove this completely in some other pr, as discussed in some other comment below?

Reply:
we still need it for the ApiConfig store

Reply:
oh wait nvm yeah